aspnet/Minitwit.Test/Repositories/MessageRepositoryTest.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+using Minitwit.Models.Context;
+using Minitwit.Models.Entity;
+using Minitwit.Repositories;
+using Minitwit.Test.Context;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+using System.Linq;
+
+namespace Minitwit.Test.Repositories
+{
+    public class MessageRepositoryTest : IDisposable
+    {
+        private readonly SqliteConnection _connection;
+        private readonly MinitwitContext _context;
+        private readonly MessageRepository _repository;
+        private readonly UserRepository _userRepository;
+
+        public MessageRepositoryTest()
+        {
+            _connection = new SqliteConnection("DataSource=:memory:");
+            _connection.Open();
+            var builder = new DbContextOptionsBuilder<MinitwitContext>().UseSqlite(_connection);
+            _context = new MinitwitTestContext(builder.Options);
+            _context.Database.EnsureCreated();
+            _repository = new MessageRepository(_context);
+            _userRepository = new UserRepository(_context);
+        }
+
+        [Fact]
+        public async Task Test_get_private_timeline()
+        {
+            var follows = (await _userRepository.GetUserFollows(2)).Select(f => f.FolloweeId).ToList();
+            var timeline = await _repository.GetPrivateTimeline(2, follows);
+            Assert.Equal(2, timeline.Count);
+        }
+
+        [Fact]
+        public async Task Test_get_messages_by_author_id()
+        {
+            var messages = await _repository.GetMessagesByAuthorId(3);
+            Assert.Equal(2, messages.Count);
```

Comment:
FluentAssertions are better IMHO
```suggestion
            messages.Should().Be(2);
```

Reply:
I disagree

Reply:
I disagree with your disagreement, this is a perfect example of internalized 4head think.
Clearly this is superior, as the structure is much closer to a natural sentence, therefore it is literally easier to read.

Reply:
I do agree that it seems easier to read, so i'll go with the fluent assertions for now.

Reply:
Changed to fluent assertion